src/EditorFeatures/Core/Implementation/CodeDefinitionWindow/CodeDefinitionWindowLocation.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.CodeDefinitionWindow
+{
+    internal struct CodeDefinitionWindowLocation
+    {
+        public string DisplayName { get; }
+        public string FilePath { get; }
+        public int Line { get; }
+        public int Character { get; }
```

Comment:
merge into a single FileLinePositionSpan?

Reply:
There's a constructor that takes a span and then drops off the end part, since we can't actually report that. We don't seem to have a FileLinePositionPoint so I'll just do FilePath + LinePosition.